platform-includes/getting-started-install/java.spring.mdx

```diff
@@ -8,20 +8,14 @@ plugins {
 }
 ```
 
-```xml {tabTitle:Maven (Spring 5)}
```

Comment:
`m` We still have the "raw" dependencies for Gradle, we could also keep them for Maven.